tests/Integration/HopTest.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen API Library for PHP
+ *
+ * Copyright (c) 2020 Adyen B.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ *
+ */
+
+namespace Adyen\Tests\Integration;
+
+use Adyen\Tests\TestCase;
+
+class HopTest extends TestCase
+{
+    public function testGetOnboardingUrl()
+    {
+        $client = $this->createClient();
+        $accountService = new \Adyen\Service\Account($client);
+        $hopService = new \Adyen\Service\Hop($client);
+
+        /* Test account holder for the HOP getOnboardingUrl test */
+        $params = $this->getTestAccountHolder();
+
+        /*
+         * Look for an existing TestAccountHolder
+         * Otherwise create a new one for the test
+         */
+        try {
+            $accountService->getAccountHolder($params);
```

Comment:
The return part of this method is not checked with assert and also it is not used to test the getOnboardingUrl. Could you elaborate the why this is needed for the getOnboardingUrl testing ? Thank you in advance,

Reply:
@aednzxy let us know